packages/indy-vdr/tests/helpers.ts

```diff
@@ -60,5 +61,8 @@ export async function createDidOnLedger(agent: Agent, submitterDid: string) {
     )
   }
 
+  // Wait some time pass to let ledger settle the object
+  await sleep(1000)
```

Comment:
Wouldn't it be better to see if we can resolve it? This would still make it non-deterministic albeit less.

Reply:
Could you elaborate a bit more on this suggestion? Do you mean to do a call to dids.resolve() before leaving this method?